packages/components/src/community/sidebar-community/sidebar-community.tsx

```diff
@@ -76,12 +75,19 @@ const SidebarCommunity = (props: Props) => {
             </Text>
             <Text size={15}>{description}</Text>
           </Stack>
-          <Stack flexDirection="row" alignItems="center" mb={12} space={8}>
-            <GroupIcon color="$neutral-100" />
+          <Stack
+            flexDirection="row"
+            alignItems="center"
+            marginBottom={12}
+            gap={4}
```

Comment:
Different gap than space before, is that correct?

Reply:
good eye, but this one is on purpose 😅